src/main/java/me/totalfreedom/totalfreedommod/rank/Title.java

```diff
@@ -7,6 +7,7 @@
 {
 
     DEVELOPER("a", "Developer", ChatColor.DARK_PURPLE, "Dev"),
+    FOUNDER("the", "Founder", ChatColor.DARK_RED, "TF Founder"),
```

Comment:
This should be "Server Founder" as this plugin does also get used on other servers, or even just "The Founder"